adapter/internal/oasparser/model/swagger.go

```diff
@@ -81,8 +80,8 @@ func (swagger *MgwSwagger) SetInfoSwagger(swagger2 spec.Swagger) error {
 	return nil
 }
 
-// setResourcesSwagger sets swagger (openapi v2) paths as mgwSwagger resources.
-func setResourcesSwagger(swagger2 spec.Swagger, isMockedAPI bool) []*Resource {
+// getResourcesSwagger sets swagger (openapi v2) paths as mgwSwagger resources.
```

Comment:
seems this function does more of unmarshalling(storable swagger to internal mgw resource obj) task rather than a simple get. how about renaming to `unmarshalSwaggerResources`

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2635/commits/58fe8546c29ba231eca3172df82e118d9194cbd8